tests/pom.xml

```diff
@@ -78,6 +80,83 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Similarly , I am not sure how all these dependencies appeared - I only added transactions and JNDI